### PR TITLE
Update reindex method for Zammad 5.2

### PR DESCRIPTION
**Pull request by MrGeneration**
Zammads searchindex-rebuild command changes with 5.2 - this PR updates the documentation accordingly.
It also adds a small hint on the update pages to make administrators  more aware.

### CODE REVIEW

**Comment by MrGeneration** [comment]
I am so sorry I have no idea why it took other commits into this PR...

File: appendix/troubleshooting/index.rst [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+      # force zammad to drop and rebuild the elasticsearch indexes
+      $ zammad run rake zammad:searchindex:rebuild
+
+   .. note:: 
+      
+      This should start rebuilding the indexes and output it's progress:
+
+      .. code-block:: sh
+
+         drop indexes...done
```
Comment (mgruner) [inline appendix/troubleshooting/index.rst:96]: The output changed slightly with https://github.com/zammad/zammad/commit/f678083c3563ebb8538e85268fcd959f9068b6a2. Please run it again with the latest state.

File: install/update.rst [hunk 2]
```diff
@@ -145,10 +145,17 @@ Updating Zammad
             $ rake db:migrate
             $ rake assets:precompile
 
-      Step 7: Start Zammad services
+      Step 7: Synchronize Zammads translation files
```
Comment (mgruner) [inline install/update.rst:148]: Typo: `Zammad's`

**Review by mgruner — APPROVED** [review]
Please see my hints.

**Pull request merged**